trade_remedies_caseworker/core/utils.py

```diff
@@ -196,16 +197,17 @@ def notify_footer(api_client, email=None):
     return default_footer
 
 
-def notify_contact_email(api_client, case_number=None):
+def notify_contact_email(api_client: Client(), case_number: uuid = None, notify_sys_param_name: uuid = None):
```

Comment:
This will instantiate an instance of Client, which we want to avoid. Also, please add the expected type of whatever the function returns (I think it's a string?)

```suggestion
def notify_contact_email(api_client: Client, case_number: uuid = None, notify_sys_param_name: uuid = None) -> str:
```